client/components/jetpack/fix-all-threats-dialog/index.tsx

```diff
@@ -49,6 +49,21 @@ const FixAllThreatsDialog = ( { onConfirmation, onCloseDialog, showDialog, threa
 		[ onCloseDialog, onConfirmation ]
 	);
 
+	const fixableThreats = useMemo(
+		() =>
+			threats.filter(
+				( threat ) => threat.fixable && threat.fixable.extras?.is_bulk_fixable !== false
+			),
+		[ threats ]
+	);
+	const nonFixableThreats = useMemo(
+		() =>
+			threats.filter(
+				( threat ) => ! threat.fixable || threat.fixable.extras?.is_bulk_fixable === false
+			),
+		[ threats ]
+	);
+
```

Comment:
If I understand correctly, we're already filtering the fixable threats [here](https://github.com/Automattic/wp-calypso/blob/10d0745a6623f7346f74b9842c92c66d296de728/client/components/jetpack/scan-threats/index.tsx#L127). The name might be a bit misleading, since this time it refers to filtering threats that can be fixed in bulk, although technically all of them can be fixed.

I'd prefer not to use `useMemo` for these, as the list is already filtered in the parent component and the computation doesn't seem expensive.

Reply:
Renamed the variables to be more precise in 54c10db55646e83a5f76f41780298b6df1a0c738.